### ISSUE TITLE
Editable Markdown Blocks

### ISSUE THREAD

**Issue by reform-chipper** [wavetermdev/waveterm#417]
Allow making a command block a dedicated markdown block for documentation purposes (not processes by the shell). This would greatly help with documenting the steps.

The broader goal is to have something like a Jupyter Notebook for shell commands.

Simple example of this vision:

![image](https://github.com/wavetermdev/waveterm/assets/162800380/8ae0efe7-429a-453b-83c9-be59e6f0b896)

**Comment by sawka** [comment]
Yes!  Been wanting to add this.  Really helps with clean import/export of "playbooks".

**Comment by David-Development** [comment]
warp.dev has this feature (part of [Warp Drive](https://www.warp.dev/warp-drive)). Would be awesome to see something similar in waveterm. I saw that it has been marked as legacy.. is this not on the roadmap anymore?
